manager/controllers/app/m4dapplication_controller_unit_test.go

```diff
@@ -813,3 +813,39 @@ func TestSyncWithPlotter(t *testing.T) {
 	g.Expect(getErrorMessages(newApp)).NotTo(gomega.BeEmpty())
 	g.Expect(newApp.Status.Ready).NotTo(gomega.BeTrue())
 }
+
+// This test checks that an empty m4dapplication can be created and reconciled
+func TestNoData(t *testing.T) {
```

Comment:
Suggest to rename to `TestM4DApplicationWithNoDatasets`

Reply:
Renamed.